pymc3/model.py

```diff
@@ -1054,6 +1054,113 @@ def _repr_latex_(self, name=None, dist=None):
 
     __latex__ = _repr_latex_
 
+    def predict(self, trace, dict_new_values, **kwargs):
```

Comment:
it sort of feels like this should _be_ `sample_posterior_predictive`, but I am ok merging like this first, and thinking about how to do that.